Wrappers/Python/test/test_algorithms.py

```diff
@@ -168,7 +168,7 @@ def test_CGLS(self):
         #ig = ImageGeometry(124,153,154)
         ig = ImageGeometry(10,2)
         numpy.random.seed(2)
-        initial = ig.allocate(0.)
+        initial = ig.allocate(1.)
```

Comment:
this change is very suspicious

Reply:
I want to be sure that the `initial` is copied into the solution.